soc/arm/nuvoton_npcx/Kconfig

```diff
@@ -20,6 +20,27 @@ menuconfig NPCX_HEADER
 
 if NPCX_HEADER
 
+config NPCX_IMAGE_NAME
```

Comment:
Is this necessary, or should we always just use the same name?

Reply:
This name is used to create the output file generated by ecst.py.  It should be different from ${KERNEL_BIN_NAME}, which defaults to zephyr.bin.  
What about setting up a default extension instead? Something like ${CONFIG_KERNEL_BIN_NAME}.npcx.bin.

Reply:
I'm OK to use CONFIG_KERNEL_BIN_NAME. Please also check the modified CL for chromium ec project: 'https://crrev.com/c/2909957'.